src/views/Languages/index.jsx

```diff
@@ -216,7 +204,108 @@ export default class Languages extends Component {
       })
     );
 
-    this.setState({ githubLoading: false });
+    if (bugzillaData) await this.fetchBugzilla(language);
+
+    this.setState({ isNextPageLoading: false });
```

Comment:
It shows up at the top of the page, so since you scroll down, you won't see it unless you scroll back up. I do realize this isn't ideal but I wasn't sure if I should change the spinner code to overlap the table and stick to the top of the view.

Reply:
Ah, ok I understand now. For a better user experience, I think we should show the spinner in the bottom when the list is loading. Have you had a look at https://github.com/bvaughn/react-virtualized/blob/master/docs/creatingAnInfiniteLoadingList.md? It looks like we can integrate this with react-virtualized.

Reply:
Didn't use that method but I believe I achieved the same effect.